connect-example/src/main/java/com/stripe/android/connect/example/ui/common/BasicExampleComponentActivity.kt

```diff
@@ -73,15 +87,42 @@ abstract class BasicExampleComponentActivity : FragmentActivity() {
         }
     }
 
-    @OptIn(ExperimentalMaterialApi::class)
     @Composable
     private fun ExampleComponentContent(
         viewModel: EmbeddedComponentLoaderViewModel,
+        edgeToEdgeEnabled: Boolean,
         openSettings: () -> Unit,
     ) {
         val state by viewModel.state.collectAsState()
         val embeddedComponentAsync = state.embeddedComponentManagerAsync
 
+        if (edgeToEdgeEnabled) {
+            // don't render the scaffold if edge-to-edge is enabled so that we get an entirely
+            // full-screen experience
+            ExampleComponentView(
```

Comment:
OK in the interim, but I think whether to show the app bar should be its own separate setting (like on iOS).

Reply:
Agreed